sessionctx/variable/session.go

```diff
@@ -352,6 +352,9 @@ type SessionVars struct {
 	// TODO: remove this after tidb-server configuration "enable-streaming' removed.
 	EnableStreaming bool
 
+	// EnableArrow indicates whether the coprocessor request can use arrow API.
+	EnableArrow bool
```

Comment:
We can make this config item be able to be reloaded without restarting the tidb-server like other hot-reloadable config items. It's not urgent, we can do this in another PR.